app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -1067,6 +1067,16 @@ define([
                 return new RegExp(param).test(value);
             },
             $.mage.__('This link is not allowed.')
+        ],
```

Comment:
I'm not super familiar with this section of the code - what's the reason we have the same implementation in `validation/rules.js` and `js/validation.js`?

Reply:
Hi @DrewML 
The validation/rules.js is only used on the admin aprt. I've created the js/validation.js so I could add the validation on the frontend also.